src/modules/[chain]/tx/[hash].vue

```diff
@@ -55,12 +55,12 @@ const messages = computed(() => {
                     tx.tx_response.code === 0 ? 'success' : 'error'
                   }`"
                 >
-                  <span
+                <VCard
```

Comment:
We don't use `VCard` any more. please change it.

Reply:
Okay